rbac/server/api/auth.py

```diff
@@ -71,6 +72,36 @@ def decorator(func):
 
 
 @AUTH_BP.post("api/authorization")
+@doc.summary("API Endpoint to authenticate and login to the NEXT platform.")
+@doc.description("API Endpoint to authenticate and login to the NEXT platform.")
+@doc.consumes(
+    doc.JsonBody(
+        {"id": str, "password": str}, description="Username and password of user"
+    ),
+    content_type="application/json",
+    location="body",
+    required=True,
+)
+@doc.response(
```

Comment:
Should we be using `@doc.produces` instead for 200 responses like this?

Reply:
Yeah - modified it to use produces.